import/popups/PopupOpener.qml

```diff
@@ -78,19 +80,13 @@ Timer {
         var winId = data.winId
         switch (topic) {
         case "embed:alert": {
```

Comment:
Can this get called more than 2 times? Should there be a queue of data objects received from the embedlite?

Reply:
This won't be called simultaneously so no queue needed. Fix is only needed to wait for the previous dialog/page to fully close.